tests/testthat/test-all.R

```diff
@@ -234,13 +234,13 @@ test_that("Read URL provided by data.json from ls.socrata() - JSON", {
   expect_equal(9, ncol(df), label="columns")
 })
 
-test_that("Read data with missing dates", { # See issue #24 & #27 
-  # Query below will pull Boston's 311 requests from early July 2011. Contains NA dates.
-  df <- read.socrata("https://data.cityofboston.gov/resource/awu8-dc52.csv?$where=case_enquiry_id< 101000295717")
-  expect_equal(99, nrow(df), label="rows")
-  na_time_rows <- df[is.na(df$TARGET_DT), ]
-  expect_equal(33, length(na_time_rows), label="rows with missing TARGET_DT dates")
-})
+# test_that("Read data with missing dates", { # See issue #24 & #27 
```

Comment:
Note: we could 
 - comment this to explain that the Boston data set we were using is no longer available
 - work to find a another similar test?

However this is fine as it stands

Reply:
Yeah, good point, we should make a reference to the issue instead of just commenting-out. Will make the change.